### PR TITLE
feat(mix-panel): add support for multiple group key value

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## Description of the change

remove [] from groupKeyVal to send it as a string value without being enclosed in [ ].

## Type of change
- [ ✅ ]Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/mp/transform.js [hunk 1]
```diff
@@ -331,7 +331,7 @@ const processGroupEvents = (message, type, destination) => {
           $token: destination.Config.token,
           $distinct_id: message.userId || message.anonymousId,
           $set: {
-            [groupKey]: [groupKeyVal]
+            [groupKey]: groupKeyVal
```
Comment (krishna2020) [inline src/v0/destinations/mp/transform.js:334]: Have we done the impact analysis ?

Comment (ujjwal-ab) [reply src/v0/destinations/mp/transform.js:334]: Yes, actually suppose for `groupKey` `company` if value `RudderStack` was passed it was showing up as `[RudderStack]` in mix panel dashboard which should not be the case. Ref:  [here](https://rudderlabs.slack.com/archives/C03ESKCMASF/p1673057546171749)

File: test/__tests__/data/mp_input.json [hunk 2]
```diff
@@ -1302,7 +1302,7 @@
       "sentat": "2020-01-24t06:29:02.359z",
       "timestamp": "2020-01-24t11:59:02.402+05:30",
       "type": "group",
-      "userid": "hjikl"
+      "userId": "hjikl"
```
Comment (ItsSudip) [inline test/__tests__/data/mp_input.json:1305]: Can you add a test case with more than one groupKeyVal?

Comment (ujjwal-ab) [reply test/__tests__/data/mp_input.json:1378]: Added

**Review by ItsSudip — DISMISSED** [review]

**Review by sandeepdsvs — DISMISSED** [review]

**Review by sandeepdsvs — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**